lxc/storage_volume.go

```diff
@@ -1340,8 +1341,9 @@ func (c *cmdStorageVolumeList) Command() *cobra.Command {
 	cmd.Aliases = []string{"ls"}
 	cmd.Short = i18n.G("List storage volumes")
 
-	c.defaultColumns = "tndcuL"
+	c.defaultColumns = "ptndcuL"
```

Comment:
The new column should be mentioned in the help.
We may also want to use `e` to line it up with `lxc list`?

Reply:
@stgraber I've remove the "Default column layout is: tndcuL" bit (as this wasn't present in `lxc list` and is already provided in the default value for the `-c` flag option. This saves having to maintain it in 2 places.

I've rearranged the documented column names into alphabetical order (like `lxc list` has them).
However `lxc list` is missing a letter column for "Project", and both "p" and "P" are taken for other things.
So we need to pick a letter for Project on "lxc list" and then use that with "lxc storage volume list" as well.

Reply:
Oh no, sorry I missed it, its "e" for "Project name". OK will use that here too.